projects/demo/src/app/app.module.ts

```diff
@@ -1,6 +1,6 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-import { OxfordModule } from 'ngx-oxford';
+import { OxfordModule } from '../../../ngx-oxford/src/lib/oxford.module';
```

Comment:
I'd prefer keeping the shorthand package name instead of relative path. It's more readable and demos how to import it as a package.